### ISSUE TITLE
Other User Mix-up

### ISSUE THREAD

**Issue by GoldenGit64** [Heath123/discross#33]
I sent a message in a server, the message took a while to send, but when it came through, it was shown to be sent by a different user.

I won’t say their username, because… privacy and minding my own business. The person who was shown is not in the server.

I’m using the Wii U web browser if that helps. (Probably doesn’t)

**Comment by Heath123** [comment]
This seems difficult to track down… is there some way to reliably reproduce it?

**Comment by GoldenGit64** [comment]
> This seems like it would be difficult to track down… is there some way to reliably reproduce it?

It had only happened that one time, and I did not do anything crazy, just a normal message send, so I am not sure how it could be reproduced. :(

**Comment by TwystedXYZ** [comment]
I have an idea about this. When adding the bot, the first message sent with it is usually the glitched user, but rectifies itself immediately after.

**Comment by GoldenGit64** [comment]
No, No. I had sent messages before with Discross in the server.

**Comment by koboshchan** [comment]
https://github.com/larsenv/discross
